src/components/animated/_liquid.less

```diff
@@ -0,0 +1,93 @@
+.liquid-btn {
+  position: relative;
+  z-index: -1;
+
+  &::before {
+    content: "";
+    width: 400px;
+    height: 400px;
+    position: absolute;
+    top: -80px;
+    left: 50%;
+    transform: translate(-50%, -75%);
+    border-radius: 45%;
+    background: rgba(255, 255, 255, 1);
+    animation: waterEffect 5s linear infinite;
+    z-index: -1;
+  }
+
+  &::after {
+    content: "";
+    width: 400px;
+    height: 400px;
+    position: absolute;
+    top: -80px;
+    left: 50%;
+    transform: translate(-50%, -75%);
+    border-radius: 40%;
+    background: rgba(255, 255, 255, 0.5);
+    animation: waterEffect 10s linear infinite;
+    z-index: -1;
+  }
+
+  .liquidBtn(@color) {
+    border-color: @color;
+    color: #000 !important;
+    .bc(@color);
+
+    &:hover::before {
+      transition: 0.5s linear;
+      top: -120px;
+    }
+
+    &:hover::after {
+      transition: 0.5s linear;
+      top: -120px;
+    }
+  }
+
+  &.blue-btn {
+    .liquidBtn(@blue);
+  }
+  &.pink-btn {
+    .liquidBtn(@pink);
+  }
+
+  &.green-btn {
+    .liquidBtn(@green);
+  }
+
+  &.yellow-btn {
+    .liquidBtn(@yellow);
+    color: #fff;
+  }
+
+  &.orange-btn {
+    .liquidBtn(@orange);
+  }
+
+  &.red-btn {
+    .liquidBtn(@red);
+  }
+
+  &.purple-btn {
+    .liquidBtn(@purple);
+  }
+
+  &.black-btn {
+    .liquidBtn(@black);
+    color: #fff !important;
+  }
+
+  &.white-btn {
+    .liquidBtn(@white);
```

Comment:
Set the background of the button transparent for white so that you can actually see the animation.

Reply:
Ok i will do these changes

Reply:
Setting this to transparent the button will look like this https://i.ibb.co/v4WMjJr/Screenshot-2020-10-04-at-6-04-34-PM.png and also wll look like this in light mode https://i.ibb.co/SNJZyCY/Screenshot-2020-10-04-at-6-05-37-PM.png.
So tell me if to retain changes and also checkout if to keep the other buttons style as this or previous one with white floating part.
In this i had made the floating part transparent and according to me it's more attractable than before. If, you say i can keep these changes or i can revert to previous one's.

Reply:
https://i.ibb.co/SNJZyCY/Screenshot-2020-10-04-at-6-05-37-PM.png

Reply:
if i do these changes the white button will like this in light mode https://i.ibb.co/SNJZyCY/Screenshot-2020-10-04-at-6-05-37-PM.png and like this in dark mode https://i.ibb.co/v4WMjJr/Screenshot-2020-10-04-at-6-04-34-PM.png.
Tell me if to keep these changes.
Also check the other buttons, i had changed the floating part to transparent rather than keeping it white. According to me this looks way more cool and attractive. Tell me if to keep these changes or revert back to previous one's.